<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamesup</class>
 <widget class="QWidget" name="gamesup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>369</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferDefault</stylestrategy>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>打开音频</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>开启视频</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>结束游戏</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>聊天窗口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>发送表情</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ChatLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ChatMsg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>241</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>房间号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="Userlist">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>121</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>在线用户：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>XX已离线..</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>XX正在讲话...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
